tests/test_match.py

```diff
@@ -280,3 +280,53 @@ def test_png_with_16_bits_per_channel():
     assert stbt.match(
         "tests/uint16.png",
         frame=cv2.imread(_find_file("uint8.png")))
+
+
+@requires_opencv_3
+def test_match_fast_path():
```

Comment:
This only requires OpenCV 3 because we complain if the image has transparency, before we decide to go down the fast path.